backend/tracim_backend/applications/collaborative_document_edition/lib.py

```diff
@@ -69,19 +81,27 @@ def _get_template_list(self) -> typing.List[str]:
         """
         return list of templates names as string like "text.odt"
         """
-        template_list = []
         try:
             is_dir_exist(self._config.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR)
             is_dir_readable(self._config.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR)
         except (NotADirectoryError) as exc:
             raise FileTemplateNotAvailable from exc
-        for filename in os.listdir(self._config.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR):
-            if not isfile(
-                join(self._config.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR, filename)
-            ):
-                continue
-            template_list.append(filename)
-        return template_list
+
+        template_filenames = [
+            entry
+            for entry in os.listdir(self._config.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR)
+            if isfile(join(self._config.COLLABORATIVE_DOCUMENT_EDITION__FILE_TEMPLATE_DIR, entry))
+        ]
+
+        if not self._config.COLLABORATIVE_DOCUMENT_EDITION__ENABLED_EXTENSIONS:
+            return template_filenames
+
+        return [
+            filename
+            for filename in template_filenames
+            if os.path.splitext(filename)[1][1:]
```

Comment:
What happen if one file has no extension here ?

Reply:
`splitext()[1]` would return an empty string, so the file wouldn't be taken into account, which is logical I think.

Reply:
ok.